### PR TITLE
Update UserMenu and PlatformSwitch

### PR DESCRIPTION
**Pull request by camilledel**
Voir #564 

TODO:

- [x] Changelog

### CODE REVIEW

File: KARL_CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Fix: Upadate `UserMenu` and `PlateformSwitch` components with `dropdown` component.
```
Comment (sunny) [inline KARL_CHANGELOG.md]: `PlatformSwitch` plutôt.

Comment (sunny) [inline KARL_CHANGELOG.md]: Et `Update`.

Comment (camilledel) [inline KARL_CHANGELOG.md]: Ahahhaha :fearful:

Comment (sunny) [inline KARL_CHANGELOG.md]: :p

**Pull request merged**